src/DistortableImageOverlay.js

```diff
@@ -274,6 +274,21 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
     return this;
   },
 
+  getAngle: function() {
+    var matrix = this._image.style.transform
```

Comment:
small refactor to use Leaflet helper methods:

```suggestion
    var matrix = L.DomUtil.getStyle(this.getElement(), 'transform')
```